collector/plugins/sources/prometheus/point_builder.go

```diff
@@ -54,7 +54,8 @@ func (builder *pointBuilder) build(metricFamilies map[string]*prom.MetricFamily)
 			if mf.GetType() == prom.MetricType_SUMMARY {
 				result = append(result, builder.buildSummaryPoints(metricName, m, now, builder.buildTags(m))...)
 			} else if mf.GetType() == prom.MetricType_HISTOGRAM {
-				if experimental.IsEnabled(experimental.HistogramConversion) {
+				// always convert control plane metrics
+				if isControlPlaneMetric(metricName) || experimental.IsEnabled(experimental.HistogramConversion) {
```

Comment:
I don't think the underlying prometheus metric name has the kubernetes_* prefix, not to mention users can change the prefix as well.  

A better approach could be to add the ability to turn on wf histograms as part of the promethues source configuration and turn it on for the control plane prometheus sources

I would also like to see a integration test for this change as well. Happy to help with that